tests/xtro-sharpie/api-annotations-dotnet/common-SharedWithYouCore.ignore

```diff
@@ -0,0 +1,2 @@
+!missing-field! SharedWithYouCoreVersionNumber not bound
+!missing-field! SharedWithYouCoreVersionString not bound
```

Comment:
Is there a reason for these?

Reply:
yes, we never bind the version string or number. If you check the ignore files:
```
git grep -rn "VersionString not bound" .
```